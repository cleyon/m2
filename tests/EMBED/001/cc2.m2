@newcmd greet{who}
@newcmd private{name}
Hello, @name@
@endcmd
@private{@{who}}
@endcmd
@@
@greet{fancy-world}
@dump commands
