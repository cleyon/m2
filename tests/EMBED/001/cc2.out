Hello, fancy-world
@newcmd greet{who}
@newcmd private{name}
Hello, @name@
@endcmd
@private{@{who}}
@endcmd
